tcn_hpl/data/vectorize/locs_and_confs.py

```diff
@@ -0,0 +1,141 @@
+import functools
+import typing as tg
+
+import numpy as np
+from numpy import typing as npt
+
+from tcn_hpl.data.vectorize._interface import Vectorize, FrameData
+from tcn_hpl.data.vectorize_classic import (
+    obj_det2d_set_to_feature,
+    zero_joint_offset,
+    HAND_STR_LEFT,
+    HAND_STR_RIGHT,
+)
```

Comment:
These don't seem to be used and can be removed?

Reply:
Correct. Removed